### ISSUE TITLE
One click duplicate for Runbooks and Reactions

### ISSUE THREAD

**Issue by hubdotcom** [Runbook/runbook#381]
Provide a duplicate button that copies all the values of a current Runbook / Reaction and opens the editor to adjust those values. Perfect for when you want to change a variable if a service is down and restore it when it's back up.

**Comment by madflojo** [comment]
Agreed! In fact we also have a few open issues that this is really the parent of.

- [ ] #333
- [ ] #332 
- [ ] #377

I think it would make sense to do a copy for monitors and reactions separately. Mainly because Reactions can be attached to multiple Runbooks. I think the export/import would be the best for tackling a situation where the whole Runbook needs to be tweaked and copied.
